version.php

```diff
@@ -25,7 +25,7 @@
 defined('MOODLE_INTERNAL') || die;
 
 $plugin->component = 'mod_board'; // Full name of the plugin (used for diagnostics).
-$plugin->version   = 2022040105; // The current module version Use 2022.04.01 as base for 4.00.
+$plugin->version   = 2022040106; // The current module version Use 2022.04.01 as base for 4.00.
```

Comment:
Version on main branch is now 2022040111, also release is '1.401.02 (Build 2022040111)';

Hoping to include this in the next release shortly, with some other PRs. TIA!